test/service/__init__.py

```diff
@@ -1,28 +1,67 @@
+import copy
+import json
 import os
+import uuid
+
+from more_itertools import flatten
 
 from app_test_case import LocalAppTestCase
 from es_test_case import ElasticsearchTestCase
-from service.data_generator.fake_data_utils import ElasticsearchFakeDataLoader
+from indexer import IndexerTestCase
 
 
-class WebServiceTestCase(ElasticsearchTestCase, LocalAppTestCase):
-    data_directory = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+class WebServiceTestCase(IndexerTestCase, LocalAppTestCase):
+    """
+    Although it seems weird for the webservice to inherit the testing mechanisms for the indexer,
+    we need them in order to send live indexer output to the webservice.
+    """
+    bundle = ("aaa96233-bf27-44c7-82df-b4dc15ad4d9d", "2018-11-02T113344.698028Z")
 
     @classmethod
     def lambda_name(cls) -> str:
         return "service"
 
-    _data_loader = None
-    seed = None  # seed is used to set a seed for the fake data loader so we can reproduce tests if needed
-    number_of_documents = 1000
-
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
-        cls._data_loader = ElasticsearchFakeDataLoader(cls.number_of_documents)
-        cls._data_loader.load_data(seed=cls.seed)
+        cls._index_canned_bundle(cls.bundle)
 
     @classmethod
     def tearDownClass(cls):
-        cls._data_loader.clean_up()
+        for index_name in cls.get_hca_indexer().index_names():
+            cls.es_client.indices.delete(index=index_name, ignore=[400, 404])
         super().tearDownClass()
+
+    @classmethod
+    def _get_documents(cls):
+        body = {
+            "query": {
+                "match_all": {}
+            }
+        }
+        return cls.es_client.search(index='azul_files_aggregate_dev', body=body)['hits']['hits']
+
+    @staticmethod
+    def _duplicate_es_doc(document):
+        """
+        takes in a template Elasticsearch document and returns a duplicate with a new entity_id
+        """
+        new_doc = copy.deepcopy(document)
+        new_id = str(uuid.uuid4())
+        new_doc['entity_id'] = new_id
+        new_doc['contents']['files'][0]['document_id'] = new_id
+        return new_doc
+
+    @classmethod
+    def _fill_index(cls, num_docs=1000):
+        """
+        Makes a bunch of copies of the first doc found in the files index and then bulk uploads them
+        """
+        template_doc = cls._get_documents()[0]['_source']
+        # -2 because the files from the template are already indexed
```

Comment:
```suggestion
        # -2 because the files from the canned bundle are already indexed
```

You should assert that there were actually two files in the bundle.

Reply:
Or, don't hard code the difference but compute it, `- len(_get_documents())`.

Reply:
Good idea. Kill the magic number